flashinfer/cute_dsl/gated_delta_rule.py

```
@@ -0,0 +1,2046 @@
+
+
+@cute.jit
+def process_first_token(
+    h_sh_chunk_curr,
+    h_chunk,
+    kq_chunk,
+    k_sh,
+    q_sh,
+    v_sh,
+    reduce_sh,
+    o_head,
```

Ruff (0.15.0)

[warning] Unused function argument: `o_head`

(ARG001)